### PR TITLE
Get ipv4 and ipv6 prop by "vm_guest_net_get"

### PR DESCRIPTION
**Pull request by sgheppy**
Enhance of "vm_guest_net_get" method:
get ipv4 and ipv6 ip Address by ipConfig object becase ipAddress attribute are deprecated

### CODE REVIEW

File: src/vpoller/vsphere/tasks.py [hunk 1]
```diff
@@ -1987,7 +1988,16 @@ def vm_guest_net_get(agent, msg):
     # Get the requested properties
     result = {}
     result['name'] = vm_name
-    result['net'] = [{prop: getattr(net, prop, '(null)') for prop in properties} for net in vm_networks]
+        result['net'] = []
```
Comment (dnaeon) [inline src/vpoller/vsphere/tasks.py]: This indentation doesn't look right.

Comment (sgheppy) [inline src/vpoller/vsphere/tasks.py]: Yes, I make a mistake.
Please ignore this pull request, I do not know well "Github" and I made a "pull request" by mistake

**Comment by dnaeon** [comment]
It seems you have accidentally replaced the `Template-VMware-vSphere-Virtual-Machine-vPoller-Native.xml` with the `Template-VMware-vSphere-Virtual-Machine-Processes-vPoller-Native.xml` template.

The first template is for Virtual Machine items, and the second one is for Processes in Virtual Machines.

Please correct this, so that I can merge your PR.

Thanks!

**Comment by sgheppy** [comment]
Thank you!
I tried to merge your master with my branch and I tried to resolve the conflicts.
I think now you can merge automatically the branch.

Thanks for your great work!

**Comment by dnaeon** [comment]
Hey Antonio,

I don't see a pull-request to the vpoller repo, so I would like to ask you
that you rebase your master branch against the vpoller/master branch and
create a pull request.

If you are having troubles with the pull-requests make sure to check
https://help.github.com/articles/creating-a-pull-request/

Thanks for your contribution!

Regards,
Marin

On Fri, May 15, 2015 at 4:23 PM, Antonio Quarta notifications@github.com
wrote:

> Thank you!
> I tried to merge your master with my branch and I tried to resolve the
> conflicts.
> I think now you can merge automatically the branch.
> 
> Thanks for your great work!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/sgheppy/py-vpoller/pull/1#issuecomment-102396526.

## 

Marin Atanasov Nikolov

dnaeon AT gmail DOT com
http://www.unix-heaven.org/

**Pull request closed without merging**